Wordle.Domain/Answer.cs

```diff
@@ -0,0 +1,34 @@
+namespace Wordle.Domain;
+
+public class Answer
+{
+    public string GetRandomWord(string filePath)
+    {
+        var wordsArr = ReadWordsFile(filePath);
+        var i = new Random().Next(wordsArr.Length);
```

Comment:
"Arr" is shorthand and exposes implementation.  This variable could jus be call `words`.